server/parsers/basic_input_output_parser.ts

```diff
@@ -13,19 +13,79 @@ const sanitize = (content: string) => {
   return DOMPurify.sanitize(content, { FORBID_TAGS: ['img'] }).replace(/!+\[/g, '[');
 };
 
+const isEveryItemString = (array?: unknown) =>
+  !!(Array.isArray(array) && array.every((item) => typeof item === 'string'));
```

Comment:
seems don't need `!!` as the result is boolean already

Reply:
Done.